Pseudocode.py

```diff
@@ -0,0 +1,64 @@
+# This pseudocode describes a process for iteratively removing the least important features from a dataset 
+# based on feature importances calculated by a model explanation method (SHAP, LIME, CitrusX). 
+# The aim is to evaluate the stability of feature importance rankings as features are removed and the model is retrained.
+# The process involves retraining a machine learning model iteratively after removing the least significant feature 
+# and calculating the consistency of feature importance rankings across iterations.
+
+
+# Initialize:
+# Model: A trained machine learning model
+# Data: Dataset used for generating explanations
+# ExplanationMethod: Method used for generating explanations (SHAP, LIME, CitrusX)
+# MinFeatures: Minimum number of features to retain
+
+# Function to compute and aggregate feature importances using a specified explanation method
+def GetFeatureImportances(Data, Model, ExplanationMethod):
+    # Compute explanations for the entire dataset using the specified method
+    # Aggregate explanations to derive feature importances
+    # Return the aggregated feature importances
+    return feature_importances
+
+# Function to calculate the consistency of feature importances across iterations
+def CalculateConsistency(OriginalImportances, NewImportances):
```

Comment:
This function should receive a list of "importances" and not only two.
Then you would apply a statistical test to measure how consistent all these importances are.
So the param should either be: 
importances: List[List[float]]
or better:
importances: np.ndarray  <-- a 2D matrix with all importances